modules/controlfreec/1.1/config/default.yaml

```diff
@@ -17,41 +17,53 @@ lcr-modules:
         options:
             configFile: "{MODSDIR}/config/freec/config_WGS.txt"
```

Comment:
Given how many of these parameters are different for WES vs. genomes, maybe some of them need to use `switch_on_wildcard()`?